zeebe/broker/src/main/java/io/camunda/zeebe/broker/partitioning/scaling/snapshot/sbe/GetSnapshotChunkSerializer.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.broker.partitioning.scaling.snapshot.sbe;
+
+import io.camunda.zeebe.broker.partitioning.scaling.snapshot.GetSnapshotChunk;
+import org.agrona.BitUtil;
+import org.agrona.MutableDirectBuffer;
+
+public class GetSnapshotChunkSerializer {
+
+  private final MessageHeaderEncoder headerEncoder = new MessageHeaderEncoder();
+  private final GetSnapshotChunkEncoder encoder = new GetSnapshotChunkEncoder();
+
+  public int size(final GetSnapshotChunk response) {
+    return MessageHeaderEncoder.ENCODED_LENGTH
+        + encoder.sbeBlockLength()
+        + 2 * (BitUtil.SIZE_OF_INT)
```

Comment:
:wrench: Why not use the standard library's `Integer.BYTES`? Applies to other serializers.